serverless/serverless
# Upgrades dependency https-proxy-agent from 1.0.0 to 2.2.0

## What did you implement:

Closes #4899 



## How did you implement it:
Upgraded https-proxy-agent from 1.0.0 to 2.2.0 to resolve DDOS vulnerability in dependency according to https://nodesecurity.io/advisories/593 


## How can we verify it:

1.) Install NSP: npm install -g nsp 
2.) Run NSP from the repo directory: nsp --check
3.) Validate that no vulnerabilities are found.

## Todos:

- [N/A] Write tests
- [N/A] Write documentation
- [N/A] Fix linting errors
- [N/A ] Provide verification config / commands / resources
- [X ] Enable "Allow edits from maintainers" for this PR
- [X ] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
